master/networking/ip-autodetection.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Configure IP auto detection
+---
+
+### Big picture
+
+Configure IP auto detection for calico nodes to ensure the correct IP address is used for routing.
+
+### Value
+
+When you install {{site.prodname}} on a node, an IP address and subnet is automatically detected using data from your datastore. {{site.prodname}} provides several ways to configure IP/subnet auto detection, and supports configuring specific IPs for:
+
+- Hosts with multiple external interfaces
+- Host interfaces with multiple IP addresses
+- Changes to subnet for cross subnet feature for packet encapsulation
+- Changes to host IP address
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **Node** resource 
+
+### Concepts
+
+#### Auto detecting node IP address and subnet
+
+For internode routing, each {{site.prodname}} node must be configured with an IPv4 address and/or an IPv6 address. When you install {{site.prodname}} on a node, a node resource is automatically created using default routing information from the datastore. For most deployments, you’ll want to update auto detection to ensure nodes get the correct IP address.
```

Comment:
FYI..I believe the guidance was from Karthik to emphasize that autodetection would not get correct address in most deployments. To ensure that users did not gloss over it and think it would automatically work. But, I did change it to your wording.